playbooks/tests/tasks/rally.yml

```diff
@@ -2,11 +2,15 @@
 - name: Running Rally against test cloud
   hosts: controller[0]
   tasks:
-    - name: copy Rally test profile template into place
+    - name: copy basic Rally test profile template into place
       copy: src=rally/bbc-cloud-validate.yml dest=./bbc-cloud-validate.yml mode=0444
 
+    - name: copy ceph Rally test profile template into place
+      copy: src=rally/bbc-cloud-validate-ceph.yml dest=./bbc-cloud-validate-ceph.yml mode=0444
+      when: cinder.enabled|bool
+
     - name: run Rally script
-      script: ./rally/run.sh {{ build_tag }}
+      script: ./rally/run.sh --cinder-enabled={{ cinder.enabled }} --build-tag={{ build_tag }}
```

Comment:
The command's arguments here do not match what the script is expecting after the change to use `getopt`